src/NuGet.Clients/NuGet.CommandLine/Commands/ProjectFactory.cs

```diff
@@ -31,6 +31,7 @@ public class ProjectFactory : MSBuildUser, IProjectFactory
 
         private Logging.ILogger _logger;
         private Configuration.ISettings _settings;
+        private bool _usingJsonFile = false;
```

Comment:
No need to init to false. Like the previous two fields don't explicitly init to null.